tv_apps/notification/js/notification.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+window.addEventListener('iac-notification-message', function(evt) {
```

Comment:
You forget to include `iac_handler.js`. So, you won't have such kind of event. If you don't want it, you can use mozSetMessageHandler('message') to handle it.